libraries/ui-library/package.json

```diff
@@ -35,17 +35,17 @@
     "codeclean": "npm run lint:fix && npm run prettier:fix"
   },
   "dependencies": {
-    "@stencil/core": "3.2.1",
+    "@stencil/core": "^4.0.3",
     "@types/resize-observer-browser": "^0.1.5"
   },
   "devDependencies": {
     "@fontsource/noto-sans": "^4.5.1",
     "@popperjs/core": "^2.5.3",
     "@stencil-community/eslint-plugin": "^0.5.0",
-    "@stencil/angular-output-target": "^0.6.0",
-    "@stencil/react-output-target": "^0.5.0",
-    "@stencil/sass": "^3.0.1",
-    "@stencil/vue-output-target": "^0.8.2",
+    "@stencil/angular-output-target": "^0.7.1",
+    "@stencil/react-output-target": "^0.5.3",
+    "@stencil/sass": "^3.0.5",
+    "@stencil/vue-output-target": "^0.8.6",
```

Comment:
Vue output target is still included, but not used in the workspaces. I would suggest we leave it and maybe in the future re-enable it